core/src/server.rs

```diff
@@ -45,3 +54,11 @@ pub async fn run_any_port(config: Config, allocated_port: oneshot::Sender<u16>)
 pub async fn run_on_port(config: Config, port: u16) {
     run(config, Some(port), None).await
 }
+
+async fn show_config(Extension(config): Extension<Config>) -> impl IntoResponse {
+    return config.serialize_pretty()
+}
+
+async fn show_version() -> impl IntoResponse {
+    return format!("Serial Keel v{}\n", env!("CARGO_PKG_VERSION"));
```

Comment:
I was thinking that `<server>/version` might be a step we do in the future as a precondition when running tests.
E.g. to check the server before tests start if the version is what we expect, and if not quit early.

Therefore the endpoint should be easily parsable.
So I suggest either just the `CARGO_PKG_VERSION` directly, or we could try this:
https://github.com/dtolnay/semver
It supports being serialized (if the feature is enabled).

So
1. Add semver as a dep
2. Parse `CARGO_PKG_VERSION` using semver
3. Do a serialized reply (e.g. JSON)

It's a bit overkill perhaps but it's also very correct 🙂